middleman/chat_functions.py

```diff
@@ -88,7 +90,16 @@ async def send_text_to_room(
         logger.exception(f"Unable to send message response to {room_id}")
         return f"Failed to send message: {ex}"
 
-
+async def get_dm(client: AsyncClient, user: str): 
+    room_id = self.store.get_dm(user_id=user)
```

Comment:
`self` wont be defined here, and neither will be `store`. So you also need to pass in `store` to this function. Which may be tricky, as the `send_text_to_room` doesn't know about `store` either.

Possibly move this logic to `Command._message` (https://github.com/elokapina/middleman/blob/a0e20a22596651cd6f5841ee36f10f002bbf527a/middleman/bot_commands.py#L80-L80), which has access to `self.store`, and then you can pass in the right room ID to `send_text_to_room` ? I think logically this is the right way to do it anyway, `send_text_to_room` is just a helper, `_message` is the method that processes the "message" command.